app/controllers/v0/map_services_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'map/security_token/service'
+
+module V0
+  class MapServicesController < SignIn::ApplicationController
+    skip_before_action :authenticate
+    before_action :load_service_account_config
+
+    # POST /v0/map_services/token
+    def token
+      result = MAP::SecurityToken::Service.new.token(
+        application: @service_account_config.service_account_id.to_sym,
```

Comment:
This is exactly the reason I'm not really sure about `service_account_id` being human readable. I think it's an unexpected usage of the id to convert it to a symbol and use it as an application identifier, and it's setting this up to be brittle and potentially an unexpected leverage of the id if we were to change it